src/compiler/scala/tools/nsc/settings/ScalaSettings.scala

```diff
@@ -254,7 +254,7 @@ trait ScalaSettings extends AbsScalaSettings
 
     private val inlineChoices = List(lMethod, inline)
     val lInline = Choice("l:inline",
-      "Enable cross-method optimizations: " + inlineChoices.mkString("", ",", "."),
+      "Enable cross-method optimizations (note: inlining requires -opt-inlnie-from): " + inlineChoices.mkString("", ",", "."),
```

Comment:
typo here. 

Just checking: do we inline unconditionally from sources in the current run, even without `-opt-inlnie-from`?

Pros: a convenient default without needing
Cons: instable results depending on separate compilation